pkg/bucketclaim/bucketclaim.go

```diff
@@ -153,9 +160,12 @@ func (b *bucketClaimListener) provisionBucketClaimOperation(ctx context.Context,
 		}
 
 		bucketClass, err := b.bucketClasses().Get(ctx, bucketClassName, metav1.GetOptions{})
-		if err != nil {
+		if kubeerrors.IsNotFound(err) {
```

Comment:
I'm not sure if `IsNotFound()` handles the nil case. Would it be better for this to be something like below, or am I missing something?

```go
if err != nil {
    if kubeerrors.IsNotFound(err) {
        b.recordEvent( ... )  
    }
    klog.V(3)....
    return err
}
```